### ISSUE TITLE
UAC functions - Auto-clean is broken

### ISSUE THREAD

**Issue by rootm0s** [rootm0s/WinPwnage#46]
- If a function fails, it does not auto-clean anymore.

**Comment by rootm0s** [comment]
After some testing, this seems to fix it. 

```
for x in Constant.output:
     if "error" in x:
	fodhelper_cleanup(path)
	return False
```

**Comment by rootm0s** [comment]
Fixed in [fb656a8](https://github.com/rootm0s/WinPwnage/commit/fb656a842f057d8944a30a4c21756cdbbdf37dcd)
